### DISCUSSION TITLE
Fortify::$registersRoutes always return false

### DISCUSSION THREAD

**Discussion by MattiaMarchiorato** [laravel/nova-issues#6633] [Q&A] [answered]
### Fortify Version

1.25.1

### Laravel Version

11

### PHP Version

8.3

### Database Driver & Version

sail83 with mysql

### Description

In our application we were using `Fortify::ignoreRoutes();` in `FortifyServiceProvider`. Due to some upgrade, we tried to use the standard Fortify routes but, deleting the `Fortify::ignoreRoutes();` the register and the others default routes were not working. After an investigation, we notice that `Fortify::$registersRoutes` is returning `false`. 

Someone have ideas why?

Thanks

### Steps To Reproduce

We tried with laravel 11 clean installation, and we don't notice the issue, we can't reproduce it.

**Comment by MattiaMarchiorato** [comment]
We notice that in the `config/app.php` inverting the order of `NovaServiceProvider` and `FortifyServiceProvider` putting first `FortifyServiceProvider` is working properly.

```
App\Providers\TelescopeServiceProvider::class,
App\Providers\FortifyServiceProvider::class,
App\Providers\NovaServiceProvider::class,
App\Providers\JetstreamServiceProvider::class,
```

**Comment by crynobone** [comment, marked as answer]
I believe this related to https://github.com/laravel/nova-issues/discussions/6619#discussioncomment-11604243
